src/services/request/RequestFactory.class.js

```diff
@@ -0,0 +1,15 @@
+const AxiosRequest = require('./AxiosRequest.class')
+const FetchRequest = require('./FetchRequest.class')
+
+class RequestFactory {
```

Comment:
I believe a class with only a single static method should not be a class. 
We should just export a function or move this logic directly in the North generic class.

Reply:
The reuqest service is also used in AliveSignal, so we can't make it North specific. I can export a function or move the static create() method into the BaseRequest class. Which one do you prefer?

Reply:
No preference.